developer-docs-site/docs/standards/fungible-asset.md

```diff
@@ -0,0 +1,451 @@
+---
+title: "Fungible Asset"
+id: "fungible-asset"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Fungible Asset
+
+Fungible assets (FA) are an essential part of the Aptos ecosystem, as they enable the creation and transfer of fungible
+units, which can represent different things, such as currency, shares, material in games, or any other type of asset.
+Fungible assets can be used to build decentralized applications that require a token economy, such as decentralized
+exchanges or gaming platforms.
+
+[Fungible asset module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move)
+provides a standard, typesafe framework for assets with fungibility.
+
+In this standard, fungible assets are stored in `Object<FungibleStore>` that has a specific amount of units, which
+can be transferred, burned, or minted. Fungible assets are units that are interchangeable with others of the same
+metadata. The standard is built upon object model so all the resources defined here are included in object resource
+group and stored inside objects.
+
+The relationship between the structures laid out in this standard is shown in this diagram.
+<div style={{textAlign: "center"}}>
+<ThemedImage
+alt="fungible asset architecture"
+sources={{
+    light: useBaseUrl('/img/docs/fungible-asset.svg'),
+    dark: useBaseUrl('/img/docs/fungible-asset-dark.svg'),
+  }}
+/>
+</div>
+
+## Difference with Aptos Coin
+
+FA is a broader category than just coins. While fungible coins are just one possible use case of FA, it can represent a wider range of fungible items, such as in-game assets, event tickets, and more. FA is constructed using an object model, which provides the flexibility for customizable, detailed management and offers a new programming model based on objects."
+The final goal of FA is to replace Aptos coin.
```

Comment:
```suggestion
FA is a broader category than just coins. While fungible coins are just one possible use case of FA, it can represent a wider range of fungible items, such as in-game assets like gems or rocks, event tickets, and partial ownership of real-world assets. FA is constructed using an object model, which provides the flexibility for customizable, detailed management and offers a new programming model based on objects.

Minimally, Aptos coin should be interchangeable with FA, it is up to the will of the ecosystem to dictate the whether or not Aptos coin is replaced by a FA equivalent.
```

Let's tone this down a bit? We haven't received deep enough feedback on the possibilities here.